doc/WebSite/Zero/Tenant-Management.md

```diff
@@ -46,13 +46,14 @@ The AbpTenant class defines some base properties, the most important are:
 -   **TenancyName**: This is the **unique** name of a tenant in the
     application. It should not normally be changed. It can be used to
     allocate subdomains to tenants like '**mytenant**.mydomain.com'.
+    As such, it cannot contain spaces.
```

Comment:
To avoid questions like:
- https://stackoverflow.com/questions/47990380/invalid-tenancy-name
- https://stackoverflow.com/questions/48316059/invalid-tenant-name-in-asp-net-boiler-plate